packages/react-i18n/src/index.tsx

```diff
@@ -70,13 +69,11 @@ export const withI18n = createHigherOrderComponent< I18nReact >( ( InnerComponen
  */
 function makeContextValue( localeData?: LocaleData ): I18nReact {
 	const i18n = createI18n( localeData );
-	const i18nLocale = localeData?.[ '' ]?.localeSlug ?? 'en';
 	return {
 		__: i18n.__.bind( i18n ),
 		_n: i18n._n.bind( i18n ),
 		_nx: i18n._nx.bind( i18n ),
 		_x: i18n._x.bind( i18n ),
 		isRTL: i18n.isRTL.bind( i18n ),
-		i18nLocale,
```

Comment:
Probably need to roll this change back and keep it around for a bit - breaking change. (and widely used in calypso) ... What's -alpha mean in the version number? Is that basically like saying this isn't semver?

Reply:
> What's -alpha mean in the version number? Is that basically like saying this isn't semver?

That's how pre-release versions are denoted according to SemVer's specs.  Ref: https://semver.org/#spec-item-9

Reply:
So safe to break backward compat here then, thanks for the link :+1:

Reply:
I'm also not sure that `@automattic/react-i18n` is used anywhere outside of a8c either so I was just planning to udpate all the uses. It looks like there is a package on the npm registry but it's private.